fea-rs/src/compile/validate.rs

```diff
@@ -44,6 +44,8 @@ pub struct ValidationCtx<'a, V: VariationInfo> {
     condition_set_defs: HashMap<SmolStr, Token>,
     aalt_referenced_features: HashMap<Tag, typed::Tag>,
     all_features: HashSet<Tag>,
+    // we cache these because they're reused a lot and slightly expensive to query
+    glyphs_app_number_idents: HashSet<SmolStr>,
```

Comment:
suggest also explaining (or linking to explanation) what the heck a glyphs app number ident is